packages/driver/src/fetchConn.ts

```diff
@@ -143,6 +143,8 @@ class BaseFetchConnection extends BaseRawConnection {
 }
 
 export class AdminUIFetchConnection extends BaseFetchConnection {
+  cliMode = true;
```

Comment:
What is the relationship between the names `cli` and `AdminUI` here? Can/should this flag be named `adminUiMode` or are there some other cli commands that would also run in this mode in the future?

Reply:
Not a blocker, for sure. Other options: `emulateCli`, `useCliBehavior`, etc. Definitely not a hard requirement and `adminUiMode` isn't much more enlightening than `cliMode` now that you've explained it.

Reply:
I think `adminUiMode` is probably the better name. I don't think we're likely to add anything else separate from the ui that will need the 'cli behaviour', and it's possible we'll need to add more hacks in the future for the ui that are not 'cli like'.